CMakePresets.json

```diff
@@ -0,0 +1,175 @@
+{
+  "version": 3,
+  "cmakeMinimumRequired": {
+    "major": 3,
+    "minor": 10,
+    "patch": 0
+  },
+  "configurePresets": [
+    {
+      "name": "default",
+      "generator": "Ninja",
+      "binaryDir": "${sourceDir}/build",
+      "cacheVariables": {
+        "CMAKE_TOOLCHAIN_FILE": {
+          "type": "FILEPATH",
+          "value": "${sourceDir}/ext_libs/vowpal_wabbit/ext_libs/vcpkg/scripts/buildsystems/vcpkg.cmake"
```

Comment:
we have submoduled vcpkg in rlclientlib, can we point to that `vcpkg.cmake` instead of vw's?